dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -167,22 +175,23 @@ public boolean isRserveConfigured() {
     }
 
     public long getUploadLogoSizeLimit() {
-        return 500000;
+        return 500_000;
```

Comment:
What kind of format is that :P? Why the change?

Reply:
https://docs.oracle.com/javase/7/docs/technotes/guides/language/underscores-literals.html
I've run pmd and it was one of the violations and I've changed it by mistake. I will get back to previous version